pynsee/macrodata/_get_insee.py

```diff
@@ -68,7 +69,7 @@ def _get_insee(api_query, sdmx_query, step="1/1"):
             list_obs.append(df)
 
         if len(list_obs) > 0:
-            obs_series = pd.concat(list_obs).reset_index(drop=True)
+            obs_series = pd.concat(list_obs)

```

Comment:
readd reset_index

Reply:
@tfardet  I may have mismanaged git (yet again), though I'm not certain I'm note missing something (black on my phone with no battery left on my laptop...). What seems like new commits should be fixes aiming to resolve conflicts with the last pr on master. I've missed your comment on the correct procedure to manage that thorough git, sorry 😑

Reply:
Ok, that one's on me; can't see how I missed the changes to master in this file, but I'll correct this asap.